runhouse/servers/cluster_servlet.py

```diff
@@ -54,17 +52,17 @@ async def __init__(
         self._key_to_env_servlet_name: Dict[Any, str] = {}
         self._auth_cache: AuthCache = AuthCache(cluster_config)
         self.autostop_helper = None
-
+        self.logger = get_logger(name=__name__, log_level=DEFAULT_LOG_LEVEL)
```

Comment:
any reason why we cant continue with the global logger import at the top of the file?